packages/eslint-plugin/docs/rules/restrict-plus-operands.md

```diff
@@ -16,12 +16,34 @@ var foo = 1n + 1;
 
 ## Options
 
+Options may be provided as an object with:
+
+- `allowAny` to ignore this rule when either left or
+  right of plus operand is a type `any`
+
 ```json
 {
-  "@typescript-eslint/restrict-plus-operands": "error"
+  "@typescript-eslint/estrict-plus-operands": [
+    "error",
+    {
+      "allowAny": true
+    }
+  ]
 }
 ```
```

Comment:
This section can be changed to just this snippet.
We try to avoid putting the full config block in rule docs now.

---

```ts
type Options = {
  allowAny?: boolean;
}

const defaultOptions: Options = {
  allowAny: false,
};
```